### ISSUE TITLE
[Feature Request] Auto export contacts when changed

### ISSUE THREAD

**Issue by mooreye** [tmo1/sms-ie#223]
If you add/remove/edit any contact, it would auto-export them.

**Comment by tmo1** [comment]
If I understand you correctly, you want SMS I/E to trigger a contacts export whenever the contact list is modified. I don't think I'm prepared to implement such a thing, but I'll leave this issue open for further discussion.

**Comment by mooreye** [comment]
> If I understand you correctly, you want SMS I/E to trigger a contacts export whenever the contact list is modified. I don't think I'm prepared to implement such a thing, but I'll leave this issue open for further discussion.

Yes, that's what I meant. I used to use SD Contacts https://f-droid.org/en/packages/cx.vmx.sdcontacts/ for this, it works fine but seems to be unmaintained.
